changelog.d/20250428_030850_shane.obrien_prepared.md

```diff
@@ -0,0 +1,40 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+### Removed
+
+- Removed support for `network-ip`. We still support `iproute`.
```

Comment:
If we remove this we de facto force a particular major version of hasql, we should update our bounds to express that

Reply:
Yes, and I've done so.

Reply:
Ah indeed I missed that